block/editor.scss

```diff
@@ -0,0 +1,5 @@
+/**
```

Comment:
Not sure if it's worth including the editor.scss if we're not using it for anything.  May as well purge it to keep clutter files down?